x-pack/plugins/endpoint/public/embeddables/resolver/store/data/graphing.test.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Store, createStore, AnyAction } from 'redux';
+import { DataAction } from './action';
+import { dataReducer } from './reducer';
+import { DataState, Vector2 } from '../../types';
+import {
+  graphableProcesses,
+  widthOfProcessSubtrees,
+  distanceBetweenNodes,
+  processNodePositionsAndEdgeLineSegments,
+} from './selectors';
+
+describe('resolver graph layout', () => {
+  let store: Store<DataState, AnyAction>;
```

Comment:
why not `ResolverAction` here?

Reply:
Changed it to `DataAction`. I forgot to change it after creating the `DataAction` type.